cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -261,10 +279,16 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   this.setCollapseTransition = setCollapseTransition;
   this.getTransition = getTransition;
   this.getDom = getDom;
+  this.isAnimating = isAnimating;
 
   // Public static properties.
   FlyoutMenu.EXPAND_TYPE = 'expand';
```

Comment:
Consider making `expand` and `collapse` constants as well. Might not make sense though if there aren't used within the file.